README.md

```diff
@@ -22,22 +22,44 @@ Interne henvendelser kan sendes via Slack i kanalen #vtp-chatten
 
 # Hvordan komme igang - Oppsett og konfigurasjon
 
-* VTP trenger Keystore og Truststore liggende i mappe .modig på brukerens hjemme-mappe, eller konfigurert gjennom verdier angitt i avsnitt om sikkerhet.
+* VTP trenger Keystore og Truststore liggende i mappe `.modig` på brukerens hjemme-mappe, eller konfigurert gjennom verdier angitt i avsnitt om sikkerhet.
 
-## Starte server fra IDE
+guide for å sette Keystore og Truststore: https://howtodoinjava.com/maven/how-to-install-maven-on-windows/
+
+* Intelliji må konfigureres med å sette classpath of module til server, og sette working directory til `$MODULE_WORKING_DIR$` i run/debug-konfiguration.
+
+## Teknologi, må være installert før du fortsetter
+* Node.js (https://nodejs.org/en/download/)
+* Yarn (https://classic.yarnpkg.com/en/docs/install/#windows-stable)
+* Java (https://www.oracle.com/java/technologies/javase-jdk11-downloads.html)
+* Maven (http://maven.apache.org/download.cgi)
+
+Etter installsjon må du legge til enviorment variabler for `MAVEN_HOME`, `JAVA_HOME`. Samt sette path i system envoirment.
```

Comment:
Oppsett av java og maven trenger vi ikke å oppgi eksplisitt. Jeg tenker at dette er del av generelt oppsett av utvikleroppsett i NAV.

Reply:
Jeg er vant til å lage en guide for at den personen med minst erfaring skal kunne sette det opp. Hvis en person som er ny skal jobbe med dette eller ikke jobbet med maven før. 

Kan fjerne det hvis det er bloat.

Reply:
Ja, tenker at vi ikke trenger å ha med oppsettet. Fint å ha med teknologiene med linker dog, også kan vi forutsette at de får installert de selv. Kunne du oppdatert Java lenken til OpenJDK også, også tenker jeg at vi er klare 👍